### ISSUE TITLE
Allow option to disable navigation

### ISSUE THREAD

**Issue by rezmeplxrf** [ueman/feedback#303]
**Is your feature request related to a problem? Please describe.**
When users navigate to another screen while being in Feedback mode, we lose the context and thus can not use any function that uses the context.

 BetterFeedback.of(context).show((feedback) async {
          if (feedback.text.isNotEmpty) {
            if (context.mounted){
               await ref.read(uploadFeedbackProvider(
                  feedback: feedback.text, screenshot: feedback.screenshot).future. // this will not be executed if user navigate during feedback mode. if we remove context.mounted then app will throw error.
              );}
            EasyLoading.showSuccess('Thanks for the feedback!');
          }
        });

**Describe the solution you'd like**
allow option to disable navigation during feedback widget is open
=

**Comment by dolbin-prime** [comment]
Yeah I think it'll be good to use an option to enable/disable the navigation mode.

The main reason I want to install the package is just to capture a screen from which an error occurs. So I want to disable the navigation mode if possible!

**Comment by M97Chahboun** [comment]
me too I need this option

**Comment by aicorey** [comment]
+1 to this
